ambry-store/src/test/java/com.github.ambry.store/BlobStoreCompactorTest.java

```diff
@@ -71,6 +71,8 @@
 
   private MetricRegistry metricRegistry;
 
+  private final ByteBuffer bundleReadBuffer = ByteBuffer.allocateDirect(100);
```

Comment:
I think this should be something like `2 * CuratedLogIndexState.PUT_RECORD_SIZE + 1` to allow for two put entries to exist on the same buffer